content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -736,15 +736,22 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-{{< feature-state for_k8s_version="v1.0" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="deprecated" >}}
 
 {{< caution >}}
-This admission controller plugin is **outdated** and **incomplete**, it may be
-unusable or not do what you would expect. It was originally designed to prevent
-the use of some, but not all, security-sensitive fields. Indeed, fields like
-`privileged`, were not filtered at creation and the plugin was not updated with
-the most recent fields, and new APIs like the `ephemeralContainers` field for a
-Pod.
+The Kubernetes project recommends that you **do not use** the
+`SecurityContextDeny` admission controller.
+
+The `SecurityContextDeny` admission controller plugin is deprecated and disabled
+by default. It will be removed in a future version. If you choose to enable the
```

Comment:
I want to check that this is a valid exception to https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future

My thought-experiment test is: if I rounded up select people from SIG Architecture and the relevant SIG, would they clearly confirm that we are unconditionally going to do that removal?

I think the answer here is Yes, but would like to check.

Reply:
Doesn't https://github.com/kubernetes/kubernetes/issues/111516 confirm that we want to remove this unconditionally? I can remove that statement if needed, it was just to address stalz feedback.

Reply:
(it's a bit stronger than “want to”; my test is “promise to remember to”)